### DISCUSSION TITLE
part-ii/2019-46/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#274] [Question comments]
Part II 2019 Differential Geometry: Paper 4, Section II, H 

https://questions.tripos.org/part-ii/2019-46/

**Comment by stevezshen** [comment]
<details>
<summary>Spoiler warning</summary>

(a) I assumed that $\gamma$ is arc length parametrised, but apparently I shouldn't do that. 

$K = -\frac{f''(v)}{f(v)}$. 

$H = \frac{1}{2}\left(-\frac{g'}{f} + f''g' - f'g''\right)$.

(c) Let $f(v) = C\cos (v)$ and $g(v) = \int^{v}_{t=0}\sqrt{1-C^2\cos^2(t)}\;\text{d}t$ for $v\in [0,\pi]$ and $C\in (0,1]$. 

</details>
